### PR TITLE
Update prettier and re-run fix-style and docs-update

### PR DESCRIPTION
**Pull request by hekike**
Update prettier and re-run fix-style and docs-update

# Changes

- update prettier and use fixed versions without `^` (`prettier` fails to follow SemVer)
- re-run fix-style
- re-run docs-update

### CODE REVIEW

File: test/server.test.js [hunk 1]
```diff
@@ -2755,9 +2743,7 @@ test('should increment/decrement inflight request count', function(t) {
     });
 });
 
-test('should increment/decrement inflight request count for concurrent reqs', function(
-    t
-) {
+test('should increment/decrement inflight request count for concurrent reqs', function(t) {
```
Comment (yunong) [inline test/server.test.js]: Pretty sure this exceeds 80 columns?

Comment (hekike) [reply test/server.test.js]: Good catch, something went wrong with this `prettier` release. I'm going to dig deeper.

**Comment by hekike** [comment]
@yunong seems to be that `prettier` cannot break lines and they changed how it breaks function parameters. I had to break them manually. I also put back the `ESLint` rule related to the line width. Now it catches long lines.

File: package.json [hunk 2]
```diff
@@ -118,26 +118,26 @@
...
+    "autocannon": "^0.16.5",
+    "autocannon-compare": "^0.3.0",
     "chai": "^4.1.2",
     "cover": "^0.2.9",
     "coveralls": "^2.13.3",
     "documentation": "^5.3.3",
-    "eslint": "^4.10.0",
-    "eslint-config-prettier": "^2.8.0",
+    "eslint": "^4.13.1",
+    "eslint-config-prettier": "2.9.0",
```
Comment (yunong) [inline package.json]: should this be floating?

File: package.json [hunk 3]
```diff
@@ -118,26 +118,26 @@
...
     "eslint-plugin-jsdoc": "^3.1.3",
-    "eslint-plugin-prettier": "^2.3.1",
+    "eslint-plugin-prettier": "2.3.1",
```
Comment (yunong) [inline package.json]: should this be floating?

File: package.json [hunk 4]
```diff
@@ -118,26 +118,26 @@
...
     "istanbul": "^0.4.5",
     "mkdirp": "^0.5.1",
     "mocha": "^3.5.3",
     "nodeunit": "^0.11.2",
     "nsp": "^2.8.1",
-    "ora": "1.3.0",
+    "ora": "^1.3.0",
     "pre-commit": "^1.2.2",
-    "prettier": "^1.8.1",
+    "prettier": "1.9.2",
```
Comment (yunong) [inline package.json]: should this be floating?

File: test/plugins/reqIdHeaders.test.js [hunk 5]
```diff
@@ -94,27 +94,28 @@ describe('request id headers', function() {
...
-    it('GH-1086: should use default uuid request id if none provided', function(
-        done
-    ) {
-        SERVER.get('/1', function(req, res, next) {
-            assert.ok(req.getId());
-            assert.ok(validator.isUUID(req.getId()));
-            res.send('hello world');
-            return next();
-        });
+    it(
```
Comment (yunong) [inline test/plugins/reqIdHeaders.test.js]: This breaking of `it` forces an extra indent. Anyway to change it so it doesn't?

File: package.json (hunk 2 above)
Comment (hekike) [reply package.json]: I did it on purpose to linter issues from prettier updates.
Whish I could comment in `package.json` :)

File: package.json (hunk 3 above)
Comment (hekike) [reply package.json]: I did it on purpose to linter issues from prettier updates.

File: test/plugins/reqIdHeaders.test.js (hunk 5 above)
Comment (hekike) [reply test/plugins/reqIdHeaders.test.js]: Unfortunately no option yet :(

**Pull request merged**